src/Pager/PagerInterface.php

```diff
@@ -30,6 +33,9 @@ public function setPage(int $page): void;
 
     public function setQuery(ProxyQueryInterface $query): void;
 
+    /**
+     * @return array&T[]
```

Comment:
array<T> does not work ?

Reply:
Not all IDEs support `T[]` type, but `array`

Reply:
I agree for `T[]` but I think that if we use `array<T>` instead it will be understood as `array` if the IDE does not support generics.

Phpstorm does not understand `array<int>` but still consider this like an `array`

Reply:
Fixes that, thanks